### ISSUE TITLE
Personal Safety (non disease), ... stay out of the hospital

### ISSUE THREAD

**Issue by weltyj** [flattenthecurve/guide#171]
Possible content to be added in the "Do's" section?

Everyone is naturally mentally distracted in this pandemic.    Please stay focused on your high risk activities, particularly driving.   Getting through your day safely so _you_ don't require emergency services or hospitalization means those limited resources can attend to COVID-19 patients.

Now, especially,  is the time to silence your cell phone when you get behind the steering wheel.

Be safe.

**Comment by jmcmurry** [comment]
Yes, same goes for any activity which is high risk but new to you. [Here's a good resource from Stanford Childrens Hospital ](https://www.stanfordchildrens.org/en/topic/default?id=sports-injury-statistics-90-P02787) outlining risks by activity.

**Comment by jmcmurry** [comment]
And don't stress infrastructure unnecessarily either. eg. [this guy destroyed his neighborhood sewage system with expanding hygroscopic beads](https://toofab.com/2020/03/02/youtuber-fills-bathtub-with-orbeez-destroys-neighborhood-sewage-system/). This is the very worst time to do so.
